config/routes.rb

```diff
@@ -8,21 +8,10 @@
   get "/", to: redirect("https://www.gov.uk/coronavirus-support-from-business")
 
   scope module: "coronavirus_form" do
-    first_question = "/medical-equipment"
+    first_question = "/accommodation"
 
     get "/start", to: redirect(path: first_question)
 
-    # Question 1.0: Can you offer medical equipment?
-    get "/medical-equipment", to: "medical_equipment#show"
-    post "/medical-equipment", to: "medical_equipment#submit"
-
-    # Question 1.2: Tell us about the medical equipment you can offer
-    get "/medical-equipment-type", to: "medical_equipment_type#show"
-    post "/medical-equipment-type", to: "medical_equipment_type#submit"
-
-    # Question 1.4: Testing equipment
-    get "/testing-equipment", to: "testing_equipment#show"
```

Comment:
As part of the A-AB-B deployment, we should probably redirect these to the accommodation question, so that anyone mid-form does not get directed to a deleted route.

Reply:
Good call - thanks @steveholmes 👍